test/runtests.bat

```diff
@@ -4,10 +4,12 @@ REM create bin directory if it doesn't exist
 if not exist ..\bin mkdir ..\bin
 
 REM compile the code into the bin folder
-javac  ..\src\seedu\addressbook\Addressbook.java -d ..\bin
+javac  ..\src\seedu\addressbook\util\FilePathUtil.java -d ..\bin\util ..\src\seedu\addressbook\Addressbook.java -d ..\bin
 
 REM run the program, feed commands from input.txt file and redirect the output to the actual.txt
 java -classpath ..\bin seedu.addressbook.AddressBook < input.txt > actual.txt
 
 REM compare the output to the expected output
-FC actual.txt expected.txt
\ No newline at end of file
+FC actual.txt expected.txt
+
+cmd /k
```

Comment:
This was added to prevent the cmd prompt window from closing so as to view the changes from FC.